### PR TITLE
[devtools-translate] Korean - What's new in DevTools (Chrome 96)

### LINKED ISSUE #1617: [devtools-translate] Korean - What's New in DevTools (Chrome 96)
Post: https://developer.chrome.com/blog/new-in-devtools-96/

Translator: @TORU0239
Reviewers: @cwdoh, @techhtml 

Stable rollout: Nov 16
Translation ready by: Nov 7

### PR DESCRIPTION
**Pull request by TORU0239**
Fixes #1617

Changes proposed in this pull request:

- Added the translation into Korean
-
-

### CODE REVIEW

**Comment by chybie** [comment]
Thanks @TORU0239 , over to @techhtml , @cwdoh, @jimoong for review!

File: site/ko/blog/new-in-devtools-96/index.md [hunk 1]
```diff
@@ -104,10 +105,11 @@ Chromium issue: [1214030](https://crbug.com/1214030)
 
 
 <!-- ## Improved the display of properties in the Properties pane {: #properties } -->
+## 속성 영역 내의 속성 표시 방법의 개선 {: #properties }
 
-<!-- The **Properties** pane now shows only relevant properties instead of showing all properties of the instance. DOM prototypes and methods are now removed. -->
+**속성** 영역은 인스턴스의 모든 속성을 표시하는 대신, 오직 관련된 속성값들만을 표시합니다. DOM 프로토타입과 메소드들은 현재 제거되었습니다.
 
-<!-- Together with the **Properties** pane [enhancements](/blog/new-in-devtools-95/#properties) in Chrome 95, you can now locate the relevant properties easier. -->
+**속성** 영역의 [크롬 95에서의 개선](/blog/new-in-devtools-95/#properties)과 함께, 여러분은 이제 관련 속성들을 쉽게 위치시킬 수 있습니다.
```
Comment (cwdoh) [inline site/ko/blog/new-in-devtools-96/index.md:112]: What about:
> 이제 손쉬운 방법으로 **속성** 영역의 [개선](/blog/new-in-devtools-95/#properties)들과 관련된 속성들을 함께 위치시킬 수 있습니다.

File: site/ko/blog/new-in-devtools-96/index.md [hunk 2]
```diff
@@ -116,13 +118,13 @@ Chromium issue: [1214030](https://crbug.com/1214030)
...
 
 
-<!-- ## Console updates -->
+## 콘솔 업데이트
 
-<!-- ### Option to hide CORS errors in the Console {: #hide-cors-errors } -->
+### 콘솔의 CORS 에러 숨김 옵션 {: #hide-cors-errors }
 
-<!-- You can hide CORS errors in the **Console**. As the CORS errors are now reported in the Issues tab, hiding CORS errors in the **Console** can help reduce the clutters. -->
+여러분은 **콘솔** 에서 CORS 오류를 숨길 수 있습니다. CORS 오류들은 현재 이슈 탭에 리포트되고 있으므로, **콘솔** 에서 이들을 숨기는 것은 콘솔 내부에 어지럽게 흩어진 오류들을 줄이는 데 도움을 줄 수 있습니다.
```
Comment (cwdoh) [inline site/ko/blog/new-in-devtools-96/index.md:125]: Removing `여러분은`. :)

File: site/ko/blog/new-in-devtools-96/index.md [hunk 3]
```diff
@@ -142,9 +144,9 @@ Chromium issue: [1251176](https://crbug.com/1251176)
 Chromium issue: [1073804](https://crbug.com/1073804)
 
 
-<!-- ### Consistent async stack traces {: #async } -->
+### 일관적인 async 스택 추적 {: #async }
 
-<!-- DevTools now reports `async` stack traces for `async` functions to be consistent with other async tasks.  -->
+다른 비동기 태스크와 일관성을 유지하기 위해, 이제 DevTools 는 `async` 함수에 대한 `async` 스택 트레이스를 보고합니다.
```
Comment (cwdoh) [inline site/ko/blog/new-in-devtools-96/index.md:149]: 보고 is right, but `보여줍니다` is more natural to me. :)

**Review by cwdoh — COMMENTED** [review]
Good job, @TORU0239. It looks done, but PTAL my suggestions. :)

File: site/ko/blog/new-in-devtools-96/index.md [hunk 4]
```diff
@@ -24,51 +24,50 @@ draft: true
...
-<!-- ## Preview feature: New CSS Overview panel {: #css-overview } -->
+## 미리보기 기능: 새로운 CSS 개요 영역 {: #css-overview }
 
-<!-- Use the new **CSS Overview** panel to identify potential CSS improvements on your page.
-[Open the **CSS Overview** panel](/docs/devtools/css-overview#open), then click on **Capture overview** to generate a report of your page’s CSS. -->
+당신의 페이지에 존재할 수 있는 잠재적 CSS 개선점을 보여 주는 새로운 CSS 개요 패널을 사용해 보세요.
+[**CSS 개요** 패널 열기](/docs/devtools/css-overview#open) 그리고 **Capture overview** 를 클릭하여 페이지 내의 CSS에 관한 보고서를 생성할 수 있습니다.
 
-<!-- You can further drill down on the information. For example, click on a color in the **Colors** section to view the list of elements that apply the same color. Click on an element to open the element in the **Elements** panel. -->
+당신은 CSS의 정보들을 더욱 깊이 파고들 수 있게 됩니다. 예를 들어, **색상** 섹션의 색상을 클릭해서 동일한 색상이 적용되는 요소들의 리스트를 볼 수 있으며, 하나의 구성요소를 클릭해서 **구성요소** 패널에서 그것을 열 수 있습니다.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-96/index.md:37]: Applied

File: site/ko/blog/new-in-devtools-96/index.md [hunk 5]
```diff
@@ -24,51 +24,50 @@ draft: true
...
 
-<!-- The **CSS Overview** panel is a preview feature. Our team is still actively working on it and we are looking for your [feedback](https://goo.gle/css-overview-feedback) for further enhancements. -->
+**CSS 개요** 패널은 미리보기 기능입니다. 우리는 이 기능에 계속적인 노력을 기울이고 있으며, 또한 더 많은 개선점을 찾기 위해서 여러분의 [피드백](https://goo.gle/css-overview-feedback) 이 필요합니다.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-96/index.md:39]: I am gonna go for 프리뷰.

File: site/ko/blog/new-in-devtools-96/index.md [hunk 6]
```diff
@@ -24,51 +24,50 @@ draft: true
...
 
 {% include 'partials/devtools/ko/banner.md' %}
 
 
-<!-- ## Preview feature: New CSS Overview panel {: #css-overview } -->
+## 미리보기 기능: 새로운 CSS 개요 영역 {: #css-overview }
 
-<!-- Use the new **CSS Overview** panel to identify potential CSS improvements on your page.
-[Open the **CSS Overview** panel](/docs/devtools/css-overview#open), then click on **Capture overview** to generate a report of your page’s CSS. -->
+당신의 페이지에 존재할 수 있는 잠재적 CSS 개선점을 보여 주는 새로운 CSS 개요 패널을 사용해 보세요.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-96/index.md:34]: Done

File: site/ko/blog/new-in-devtools-96/index.md [hunk 7]
```diff
@@ -24,51 +24,50 @@ draft: true
...
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/ef26abc89035075bbdb08f1b26c1b8fd942ffc04 #}
 
 Chromium issue: [1254557](https://crbug.com/1254557)
 
+## 렌더링 탭의 업데이트
 
-<!-- ## Rendering tab updates  -->
-
-<!-- ### Emulate the CSS prefers-contrast media feature {: #prefers-contrast } -->
+### CSS의 prefers-constrast 미디어 기능의 에뮬레이션 {: #prefers-contrast }
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-96/index.md:51]: Modified

File: site/ko/blog/new-in-devtools-96/index.md [hunk 8]
```diff
@@ -24,51 +24,50 @@ draft: true
...
+[**CSS 개요** 패널 열기](/docs/devtools/css-overview#open) 그리고 **Capture overview** 를 클릭하여 페이지 내의 CSS에 관한 보고서를 생성할 수 있습니다.
 
-<!-- You can further drill down on the information. For example, click on a color in the **Colors** section to view the list of elements that apply the same color. Click on an element to open the element in the **Elements** panel. -->
+당신은 CSS의 정보들을 더욱 깊이 파고들 수 있게 됩니다. 예를 들어, **색상** 섹션의 색상을 클릭해서 동일한 색상이 적용되는 요소들의 리스트를 볼 수 있으며, 하나의 구성요소를 클릭해서 **구성요소** 패널에서 그것을 열 수 있습니다.
 
-<!-- The **CSS Overview** panel is a preview feature. Our team is still actively working on it and we are looking for your [feedback](https://goo.gle/css-overview-feedback) for further enhancements. -->
+**CSS 개요** 패널은 미리보기 기능입니다. 우리는 이 기능에 계속적인 노력을 기울이고 있으며, 또한 더 많은 개선점을 찾기 위해서 여러분의 [피드백](https://goo.gle/css-overview-feedback) 이 필요합니다. 
 
-<!-- Read [this article](/docs/devtools/css-overview) to learn more on the **CSS Overview** panel. -->
+**CSS 개요** 에 대해 좀더 알고 싶으시다면 [더보기](/docs/devtools/css-overview) 를 클릭하여 첨부된 문서를 읽어 보세요.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-96/index.md:41]: Applied

File: site/ko/blog/new-in-devtools-96/index.md [hunk 9]
```diff
@@ -24,51 +24,50 @@ draft: true
...
 
-<!-- ## Rendering tab updates  -->
-
-<!-- ### Emulate the CSS prefers-contrast media feature {: #prefers-contrast } -->
+### CSS의 prefers-constrast 미디어 기능의 에뮬레이션 {: #prefers-contrast }
 
 {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/47fsHvVLiVC9J0eWY9wD.png", alt="Emulate the CSS prefers-contrast media feature", width="800", height="483" %}
 
-<!-- The [prefers-contrast](https://www.chromestatus.com/feature/5646323212615680) media feature is used to detect if the user has requested more or less contrast in the page. -->
+[prefers-contrast](https://www.chromestatus.com/feature/5646323212615680) 미디어 기능은 유저가 더 적거나 더 많은 대비를 요청했는지를 알아보기 위한 기능입니다.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-96/index.md:55]: Good suggestion

File: site/ko/blog/new-in-devtools-96/index.md [hunk 10]
```diff
@@ -24,51 +24,50 @@ draft: true
...
 
 
-<!-- ### Emulate the Chrome’s Auto Dark Theme feature {: #auto-dark-mode } -->
+### 크롬의 자동 어두운 테마 기능 에뮬레이션 {: #auto-dark-mode }
 
-<!-- Use DevTools to emulate auto dark theme to easily see how your page looks when Chrome’s [Auto Dark Theme](/blog/auto-dark-theme/) is enabled. -->
+DevTools 를 사용하여 자동 어두운 테마를 에뮬레이션할 수 있으며, 이를 이용해 사용자들은 크롬의 [자동 어두운 테마](/blog/auto-dark-theme/)가 설정되었을 때 어떻게 보일지를 쉽게 확인할 수 있습니다.
 
-<!-- Chrome 96 introduces an [Origin Trial](/blog/origin-trials/) for [Auto Dark Theme](/blog/auto-dark-theme/) on Android. With this feature, the browser applies an automatically generated dark theme to light themed sites, when the user has opted into dark themes in the Operating System.  -->
+크롬 96은 안드로이드의 [자동 어두운 테마](/blog/auto-dark-theme/) 에 대한 [Origin Trial](/blog/origin-llllllll/) 을 제시합니다. 이 기능을 이용하여, 사용자가 운영체제 단에서 다크 테마를 적용했을 때, 크롬 브라우저는 자동 생성된 라이트 테마를 라이트 테마로 제작된 웹사이트에 적용합니다.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-96/index.md:68]: Oh my mistake, thanks

File: site/ko/blog/new-in-devtools-96/index.md (hunk 1 above)
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-96/index.md:112]: Good

File: site/ko/blog/new-in-devtools-96/index.md (hunk 3 above)
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-96/index.md:149]: Appllied

**Comment by TORU0239** [comment]
I just updated my branch with suggestions from @cwdoh, please review it. and any objections please? @techhtml @jimoong

File: site/_includes/partials/devtools/ko/whats-new.md [hunk 11]
```diff
@@ -1,18 +1,20 @@
...
-* [New Payload tab in the Network panel](/blog/new-in-devtools-96/#payload)
-* [Improved the display of properties in the Properties pane](/blog/new-in-devtools-96/#properties)
-* [Option to hide CORS errors in the Console](/blog/new-in-devtools-96/#hide-cors-errors)
-* [Proper `Intl` objects preview and evaluation in the Console](/blog/new-in-devtools-96/#intl)
-* [Consistent async stack traces](/blog/new-in-devtools-96/#async)
-* [Retain the Console sidebar](/blog/new-in-devtools-96/#console-sidebar)
-* [[Experimental] New Reporting API pane in the Application panel](/blog/new-in-devtools-96/#reporting-api) -->
+### Chrome 96 {: #chrome96 }
+* [미리보기 기능: 새로운 CSS 개요 영역](/ko/blog/new-in-devtools-96/#css-overview)
+<!-- * [Restored and improved CSS length edit and copy experince](/ko/blog/new-in-devtools-96/#length) -->
```
Comment (chybie) [inline site/_includes/partials/devtools/ko/whats-new.md:6]: Please update this

File: site/ko/blog/new-in-devtools-96/index.md [hunk 12]
```diff
@@ -47,28 +47,48 @@ draft: true
 Chromium issue: [1254557](https://crbug.com/1254557)
 
 
-<!-- ## Rendering tab updates  -->
+<!-- ## Restored and improved CSS length edit and copy experince {: #length } -->
 
-<!-- ### Emulate the CSS prefers-contrast media feature {: #prefers-contrast } -->
```
Comment (chybie) [inline site/ko/blog/new-in-devtools-96/index.md:52]: I added a new section here due to we merged some fixes in M96. Please help to translate. Thanks!

**Review by chybie — CHANGES_REQUESTED** [review]

File: site/_includes/partials/devtools/ko/whats-new.md (hunk 11 above)
Comment (TORU0239) [reply site/_includes/partials/devtools/ko/whats-new.md]: Updated.

File: site/ko/blog/new-in-devtools-96/index.md (hunk 12 above)
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-96/index.md:52]: Added new translation for the new section

**Comment by TORU0239** [comment]
Translated newly added section @chybie Please review this, thanks! back to you, @cwdoh @techhtml @jimoong

**Comment by chybie** [comment]
다들 감사 해요!

**Review by chybie — APPROVED** [review]

**Pull request merged**